### ISSUE TITLE
Tailscale Authentification never opens

### ISSUE THREAD

**Issue by meltforge** [jens-maus/RaspberryMatic#2541]
### Describe the issue you are experiencing

After activating Tailscale in the network settings and pressing the "Authentification" button, a new browser window opens and displays "Loading..." forever. 

### Describe the behavior you expected

The new browser should open the Tailscale Login Page.

### Steps to reproduce the issue

1. Einstellungen -> Systemsteuerung -> Netzwerkeinstellungen
2.Setting "Tailscale VPN-Funktion" active
3. Pressing the button

### What is the version this bug report is based on?

3.73.9.20231130

### Which base platform are you running?

ha-addon (HomeAssistant Add-on)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
behavior was the same on previous versions and when running as standalone VM (Synology)
```

**Comment by jens-maus** [comment]
Can reproduce that issue. Seems that tailscale has changes something internally and we have to adapt/fix that in a future version. See:

<img width="809" alt="Bildschirmfoto 2023-11-30 um 18 32 40" src="https://github.com/jens-maus/RaspberryMatic/assets/5887769/914e0c44-aa4d-43d5-aac4-27dd37d8fc05">

Too bad that you did not report that earlier, because 3.73.9.20231130 was just released ;-)

**Comment by meltforge** [comment]
Yeah, well...after updating I remembered the issue and checked if it's still not working 😏

**Comment by jens-maus** [comment]
Current workaround: Restart RaspberryMatic after you have enabled Tailscale by using the checkbox. After the reboot the "Authentifizierung" button will work nicely.

**Comment by meltforge** [comment]
Confirmed! Thank you!
